apps/api/src/external/pcglAuthZClient.ts

```diff
@@ -137,3 +155,117 @@ export const getUserInformation = async (
 		return failure('SYSTEM_ERROR', `Error contacting the PCGL Authorization Service.`);
 	}
 };
+
+/**
+ * Function to lookup a user by their email address in the AuthZ service
+ * @param emailAddress
+ * @param accessToken
+ * @param retry
+ * @returns a list of PCGL IDs that match the email provided
+ */
+export const lookupUserByEmail = async (
+	emailAddress: string,
+	accessToken: string,
+	retry = 0,
+): AsyncResult<PCGLAuthzLookupUserResponse, 'SYSTEM_ERROR' | 'NOT_FOUND'> => {
+	try {
+		const queryParams = new URLSearchParams();
+		queryParams.set('email', emailAddress);
+		const response = await fetchAuthZResource(`/user/lookup?${queryParams.toString()}`, accessToken);
+
+		if (response.status === 404) {
+			const message = `No user found with email ${emailAddress}.`;
+			logger.info('[AUTHZ]:', `retry: ${retry}`, message);
```

Comment:
Should the comment specify "Retries remaining:" instead of "retry:"? As written, the log seems to indicate how many times its already been attempted.